app/src/main/java/com/nutomic/syncthingandroid/activities/FolderActivity.java

```diff
@@ -304,12 +324,18 @@ public void onPause() {
     @Override
     protected void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
-        outState.putBoolean(IS_SHOWING_DELETE_DIALOG, mDeleteDialog != null && mDeleteDialog.isShowing());
+        outState.putBoolean(IS_SHOW_DELETE_DIALOG, mDeleteDialog != null && mDeleteDialog.isShowing());
         Util.dismissDialogSafe(mDeleteDialog, this);
 
+        outState.putBoolean(IS_SHOW_OVERRIDE_CHANGES_DIALOG, mOverrideChangesDialog != null && mOverrideChangesDialog.isShowing());
+        Util.dismissDialogSafe(mOverrideChangesDialog, this);
+
         if (mIsCreateMode){
             outState.putBoolean(IS_SHOW_DISCARD_DIALOG, mDiscardDialog != null && mDiscardDialog.isShowing());
             Util.dismissDialogSafe(mDiscardDialog, this);
+
+            outState.putBoolean(IS_SHOW_OVERRIDE_CHANGES_DIALOG, mOverrideChangesDialog != null && mOverrideChangesDialog.isShowing());
+            Util.dismissDialogSafe(mOverrideChangesDialog, this);
```

Comment:
I agree, was simply following the existing code... will remove, again.

Reply:
Just in case as you removed in the other case: It's not removed yet here.  
(Also not that it matters, but here I don't see what the existing code is that is doing the same/something similar.)